src/server/pfs/server/driver_v2.go

```diff
@@ -1083,6 +1083,9 @@ func (d *driverV2) createTmpFileSet(server pfs.API_CreateTmpFileSetServer) (stri
 }
 
 func (d *driverV2) renewTmpFileSet(ctx context.Context, id string, ttl time.Duration) error {
+	if ttl.Seconds() == 0 {
```

Comment:
When we talked offline I meant to do this on the client, and keep the units as seconds. If you are changing it to nanoseconds just do `ttl < time.Second`, which is more clear.

Reply:
Refer to the comment above.